### PR TITLE
update how to get henakaku token only jp page

### PR DESCRIPTION
**Pull request by isbtty**
# What
- Discordの会話のオンボーディングを元に内容を修正

### CODE REVIEW

**Comment by isbtty** [comment]
@wataruoguchi このような形でいかがでしょうか？？

File: henkaku-token-ja.md [hunk 1]
```diff
@@ -16,11 +16,31 @@ $HENKAKU トークン(以下、トークン)は、変革会コミュニティの
...
-- コミュニティプロポーザルの作成および投票には [snapshots.org](https://snapshot.org/#/henkaku.eth/) を利用しています。
\ No newline at end of file
+  - 0 HENAKAKUと表示されればウォレットの設定は完了です
+  - トークンは$HENKAKUと呼ばれ、Polygonブロックチェーンにおけるアドレスは次の通りです:  [0xd59FFEE93A55F67CeD0F56fa4A991d4c8c8f5C4E](https://polygonscan.com/token/0xd59FFEE93A55F67CeD0F56fa4A991d4c8c8f5C4E)
+- Henkaku Discordにログインして、👋wellcomeと🚀start-hereを読んでください。案内に従うと他のチャンネルへのアクセス権を獲得することができます。
+- #memberチャンネルで挨拶をしたら、`!register` コマンドを叩いてください。
+  - Collab.LandというアカウントからDMがきます。このアカウントの案内にしたがってMetaMaskを連携してください。
+  - 注意: botと同じ名前を名乗ったscamユーザーがメッセージを送ってくる場合もあるらしいのでBOTかどうかリンクをクリックする前に目視確認をお願いします。
+  - 注意: 初期設定がDMをブロックしている可能性があるので下記を参考に許可してください。
+    - https://support.discord.com/hc/ja/articles/217916488-%E3%83%96%E3%83%AD%E3%83%83%E3%82%AF%E3%81%A8%E3%83%97%E3%83%A9%E3%82%A4%E3%83%90%E3%82%B7%E3%83%BC%E8%A8%AD%E5%AE%9A
```
Comment (wataruoguchi) [inline henkaku-token-ja.md]: nit pickingですが、Markdownのリンク記法を利用することでスッキリします。

```md
注意: 初期設定がDMをブロックしている可能性があるので[こちらのページ](https://support.discord.com/hc/ja/articles/217916488-%E3%83%96%E3%83%AD%E3%83%83%E3%82%AF%E3%81%A8%E3%83%97%E3%83%A9%E3%82%A4%E3%83%90%E3%82%B7%E3%83%BC%E8%A8%AD%E5%AE%9A)を参考に許可してください。
```

**Review by wataruoguchi — APPROVED** [review]
更新ありがとうございます！素晴らしい。英語版にも反映させたいですね

**Comment by wataruoguchi** [comment]
@TatsuyaIshibe ドキュメントチェックをしているようですのでmergeは待っていただけますか？

**Comment by wataruoguchi** [comment]
@TatsuyaIshibe 待っている間、英語でもコンテンツを作ることは可能ですか？admin channelで英語ネイティブのユーザーに英語チェックしてもらってcommitできたら最高です。もし大変でしたらdiscord上でお声がけください！私も協力します。

**Comment by isbtty** [comment]
@wataruoguchi 英語も時間見つけてやりますね！

**Comment by Joi** [comment]
Are you waiting for me to merge or will you merge when ready?

**Comment by wataruoguchi** [comment]
@Joi  the latter! If things are good either Ishibe san or I will merge :)

**Review by Joi — COMMENTED** [review]
👋wellcome -> 👋welcome
hit the `!register` command -> go to the #join channel and type `!register`.
You will get a DM from an account called Collab.Land. -> You will get a DM from an account called coinvise-bot.
:note: Some scam users with the same name as the bot may send you a message, so please check visually before clicking on the link to see if it is a bot. -> :note: Some scammers with the same name as the bot may send you a message, so please check the name of the bot and confirm it is coinvise-bot#4309 - also make sure it is the bot the messaged you right after you did the `!register` command.

**Review by Joi — CHANGES_REQUESTED** [review]
will give you a $HENAKU token -> will give can give you $HENAKU tokens

**Comment by Joi** [comment]
Actually, I'll make the changes directly in a few min.

**Review by Joi — APPROVED** [review]

**Comment by Joi** [comment]
So I edited the English but I added some new stuff so maybe it's a bit out of sync with the Japanese now. :-) Can you take a look @wataruoguchi and @TatsuyaIshibe?

**Comment by wataruoguchi** [comment]
@TatsuyaIshibe I've just updated the Japanese page based on what @Joi has changed. Would you double check and merge it, please? Good work all!

**Comment by isbtty** [comment]
LGTM!

**Pull request merged**